tests/installation/user_settings.pm

```diff
@@ -43,13 +41,17 @@ sub run() {
     # done user setup
     send_key $cmd{next};
 
-    # loading cracklib
-    # If check_screen added to workaround bsc#937012
+    # PW too easy (cracklib)
     if (check_screen('inst-userpasswdtoosimple', 13)) {
         send_key "ret";
     }
     else {
-        record_soft_failure 'bsc#937012';
+        if (check_var('LIVECD', 1)) {
+            record_soft_failure 'boo#1013206';
+        }
+        else {
+            record_soft_failure 'bsc#937012';
```

Comment:
so skip this on check_var('VERSION', '12') && check_var('ARCH', 's390x') and assert everywhere else.